src/notebook.envd

```diff
@@ -0,0 +1,9 @@
+def jupyter_lab(port):
```

Comment:
I think we should identity `local_port` and `host_port` in user interface also, and with default arguments so that user can use it out-of-the-box.

For example, 
```
# -1 can mean any port works
def jupyter_lab(host_port=-1, local_port=8888):
       """Configure Jupyter Lab.

    Args:
        port (int): port used by host and container
    """
    install.python_packages(["jupyterlab"])
    expose(local_port=local_port, host_port=host_port, svc="jupyter")
    runtime.daemon(commands=["jupyter-lab --port="+str(local_port)])
```

Reply:
LGTM. For now, only the host_port supports random choice by envd if it's 0. I will update this.

Reply:
Fixed. PTAL.